test/puppetlabs/puppetdb/http/facts_test.clj

```diff
@@ -749,6 +750,26 @@
   ([endpoint query paging-options]
    (:results (raw-query-endpoint endpoint query paging-options))))
 
+(deftestseq large-integer-storage
+  [[version endpoint] facts-endpoints
+   method [:get :post]]
+
+  (let [f {:certname "a.local" :name "large-int" :environment "DEV"
+           :value 3546565741691827463891754698375639827814623789421387401348}]
```

Comment:
Ha! I wonder what happens, when you throw a float in JSON, that is larger than what java can handle with the float primitives. Does that turn into a BigInteger, and then attempt to get stored in here?